docs/source/convert.rst

```diff
@@ -1,3 +1,5 @@
+.. _convert:
```

Comment:
What's this? Is it a labelled anchor to refer to the page more explicitly? 

FYI, this one shows up as a string in the rendered docs. The one in `roadmap.md` doesn't. I can only that it's b/c  this page is rst and the other one md.

Reply:
This is some remnant of trying to cross-reference .rst, but can be safely removed now that we are in jupyter book! I think we have whole bunch of such cross-ref things needing revision/deletion.